lib/shelley/exe/shelley-test-cluster.hs

```diff
@@ -0,0 +1,224 @@
+-- each (so 1M Ada in total). Additionally, the file also contains a set of
+-- wallets with pre-existing rewards (1M Ada) injected via MIR certificates.
+-- These wallets have the same UTxOs as other faucet wallets.
+--
+-- Some additional wallets of interest:
+--
+-- - (Shelley) Has a pre-registered stake key but no delegation.
+--
+--     [ "over", "decorate", "flock", "badge", "beauty"
+--     , "stamp", "chest", "owner", "excess", "omit"
+--     , "bid", "raccoon", "spin", "reduce", "rival"
+--     ]
+--
+-- - (Shelley) Contains only small coins (but greater than the minUTxOValue)
+--
+--     [ "either" , "flip" , "maple" , "shift" , "dismiss"
+--     , "bridge" , "sweet" , "reveal" , "green" , "tornado"
+--     , "need" , "patient" , "wall" , "stamp" , "pass"
+--     ]
+--
+-- - (Shelley) Contains 100 UTxO of 100_000 Ada, and 100 UTxO of 1 Ada
+--
+--     [ "radar", "scare", "sense", "winner", "little"
+--     , "jeans", "blue", "spell", "mystery", "sketch"
+--     , "omit", "time", "tiger", "leave", "load"
+--     ]
+--
+-- - (Icarus) Has only addresses that start at index 500
+--
+--     [ "erosion", "ahead", "vibrant", "air", "day"
+--     , "timber", "thunder", "general", "dice", "into"
+--     , "chest", "enrich", "social", "neck", "shine"
+--     ]
+--
+-- - (Byron) Has only 5 UTxOs of 1,2,3,4,5 Lovelace
+--
+--     [ "suffer", "decorate", "head", "opera"
+--     , "yellow", "debate", "visa", "fire"
+--     , "salute", "hybrid", "stone", "smart"
+--     ]
+--
+-- - (Byron) Has 200 UTxO, 100 are worth 1 Lovelace, 100 are woth 100_000 Ada.
+--
+--     [ "collect", "fold", "file", "clown"
+--     , "injury", "sun", "brass", "diet"
+--     , "exist", "spike", "behave", "clip"
+--     ]
+--
+-- - (Ledger) Created via the Ledger method for master key generation
+--
+--     [ "struggle", "section", "scissors", "siren"
+--     , "garbage", "yellow", "maximum", "finger"
+--     , "duty", "require", "mule", "earn"
+--     ]
+--
+-- - (Ledger) Created via the Ledger method for master key generation
+--
+--     [ "vague" , "wrist" , "poet" , "crazy" , "danger" , "dinner"
+--     , "grace" , "home" , "naive" , "unfold" , "april" , "exile"
+--     , "relief" , "rifle" , "ranch" , "tone" , "betray" , "wrong"
+--     ]
+--
+-- # CONFIGURATION
+--
+-- There are several environment variables that can be set to make debugging
+-- easier if needed:
+--
+-- - CARDANO_WALLET_PORT: choose a port for the API to listen on (default: random)
+--
+-- - CARDANO_NODE_TRACING_MIN_SEVERITY: increase or decrease the logging
+--                                      severity of the nodes. (default: Error)
+--
+-- - CARDANO_WALLET_TRACING_MIN_SEVERITY: increase of decrease the logging
+--                                        severity of cardano-wallet. (default: Info)
+--
+-- - NO_CLEANUP: If set, the temporary directory used as a state directory for
+--               nodes and wallet data won't be cleaned up.
```

Comment:
Why no `NO_POOLS` option for fast boot?

Reply:
Oversight :+1:

Reply:
Not sure if an additional oversight or not, but still doesn't work.

Reply:
8da9ccc